src/styles/text/text.js

```diff
@@ -305,18 +335,33 @@ Object.assign(TextStyle, {
         // Process labels by priority
         let priorities = Object.keys(labels).sort((a, b) => a - b);
         for (let priority of priorities) {
-            if (!labels[priority]) {
+            if (!labels[priority]) { // no labels at this priority, skip to next
                 continue;
             }
 
+            // i is a label object at the current priority level, example:
```

Comment:
Not sure where this is coming from, but `i` is not the label object of the current priority, nor is `labels[priority][i]`. But the object description is ok.